[
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/293844218",
    "html_url": "https://github.com/electron/electron/issues/9174#issuecomment-293844218",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/9174",
    "id": 293844218,
    "user": {
      "login": "deepak1556",
      "id": 964386,
      "avatar_url": "https://avatars2.githubusercontent.com/u/964386?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/deepak1556",
      "html_url": "https://github.com/deepak1556",
      "followers_url": "https://api.github.com/users/deepak1556/followers",
      "following_url": "https://api.github.com/users/deepak1556/following{/other_user}",
      "gists_url": "https://api.github.com/users/deepak1556/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/deepak1556/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/deepak1556/subscriptions",
      "organizations_url": "https://api.github.com/users/deepak1556/orgs",
      "repos_url": "https://api.github.com/users/deepak1556/repos",
      "events_url": "https://api.github.com/users/deepak1556/events{/privacy}",
      "received_events_url": "https://api.github.com/users/deepak1556/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2017-04-13T09:49:22Z",
    "updated_at": "2017-04-13T10:04:35Z",
    "body": "Have verified that chrome 56 reports the right value for `Network.loadingFailed` canceled but blockedReason is undefined though. If we reject the requests with `ERR_ABORTED` instead of `ERR_BLOCKED_BY_CLIENT` then the value is reported fine. Need to get some clarity from upstream on this."
  },
  {
    "url": "https://api.github.com/repos/electron/electron/issues/comments/308666324",
    "html_url": "https://github.com/electron/electron/issues/9174#issuecomment-308666324",
    "issue_url": "https://api.github.com/repos/electron/electron/issues/9174",
    "id": 308666324,
    "user": {
      "login": "zcbenz",
      "id": 639601,
      "avatar_url": "https://avatars3.githubusercontent.com/u/639601?v=3",
      "gravatar_id": "",
      "url": "https://api.github.com/users/zcbenz",
      "html_url": "https://github.com/zcbenz",
      "followers_url": "https://api.github.com/users/zcbenz/followers",
      "following_url": "https://api.github.com/users/zcbenz/following{/other_user}",
      "gists_url": "https://api.github.com/users/zcbenz/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/zcbenz/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/zcbenz/subscriptions",
      "organizations_url": "https://api.github.com/users/zcbenz/orgs",
      "repos_url": "https://api.github.com/users/zcbenz/repos",
      "events_url": "https://api.github.com/users/zcbenz/events{/privacy}",
      "received_events_url": "https://api.github.com/users/zcbenz/received_events",
      "type": "User",
      "site_admin": true
    },
    "created_at": "2017-06-15T08:34:48Z",
    "updated_at": "2017-06-15T08:34:48Z",
    "body": "Hmm Chrome also rejects with `ERR_BLOCKED_BY_CLIENT` in its webRequest API, honestly I don't know `Network.loadingFailed` should look like for cases like this, but it is probably too late to change the behavior. Closing this as won't fix."
  }
]